catalogue/webapp/next.config.js

```diff
@@ -1,4 +1,4 @@
 const webpack = require('webpack');
 const config = require('@weco/common/next/next.config');
 
-module.exports = config(webpack);
+module.exports = config(webpack, 'works');
```

Comment:
Would this make more sense as 'catalogue'?

Reply:
Probably, but the DNS is setup for now.